ext/mini_racer_extension/serde.c

```diff
@@ -240,24 +246,26 @@ static void ser_num(Ser *s, double v)
 }
 
 // ser_bigint: |n| is in bytes, not quadwords
-static void ser_bigint(Ser *s, const uint64_t *p, size_t n, int sign)
+static void ser_bigint(Ser *s, const void *p, size_t stride, size_t n, int sign)
```

Comment:
Updated to handle both 32- and 64-bit input pointers given two invocations in the code.